packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -386,7 +386,7 @@ describe("editable-tree", () => {
         };
         const forest = setupForest(schemaData, [{ type: optionalChildSchema.name, fields: { child: [{ type: int32Schema.name, value: undefined }] } }]);
         const [context, field] = getEditableTree(forest);
-        assert.throws(() => ((field as EditableTree).child), /`undefined` values not allowed for primitive fields/);
+        assert.throws(() => ((field as EditableTree).child), /Error: 0x3c7/);
```

Comment:
Verify this with tree experts.

Reply:
We should be able to use `validateAssertionError` and keep the human-friendly string here. [Example](https://github.com/microsoft/FluidFramework/blob/68c2bc5a2b4ee3ae9acb8f42c0ed966d366c4d19/packages/runtime/datastore/src/test/localChannelContext.spec.ts#L34-L36):

![image](https://user-images.githubusercontent.com/716334/190712796-1ecd89e4-b8ec-491c-a6b8-4f1ea8b95719.png)

Reply:
this seems like a misuse of assert to me. asserts should never happen. for user facing errors we should be throwing usage errors